packages/framework/fluid-static/src/test/types/validateFluidStaticPrevious.generated.ts

```diff
@@ -205,6 +205,30 @@ declare function use_old_InterfaceDeclaration_IMember(
 use_old_InterfaceDeclaration_IMember(
     get_current_InterfaceDeclaration_IMember());
 
+/*
+* Validate forward compat by using old type in place of current type
+* If breaking change required, add in package.json under typeValidation.broken:
+* "InterfaceDeclaration_IRootDataObject": {"forwardCompat": false}
+*/
+declare function get_old_InterfaceDeclaration_IRootDataObject():
+    TypeOnly<old.IRootDataObject>;
+declare function use_current_InterfaceDeclaration_IRootDataObject(
+    use: TypeOnly<current.IRootDataObject>);
+use_current_InterfaceDeclaration_IRootDataObject(
+    get_old_InterfaceDeclaration_IRootDataObject());
+
+/*
+* Validate back compat by using current type in place of old type
+* If breaking change required, add in package.json under typeValidation.broken:
+* "InterfaceDeclaration_IRootDataObject": {"backCompat": false}
+*/
+declare function get_current_InterfaceDeclaration_IRootDataObject():
+    TypeOnly<current.IRootDataObject>;
+declare function use_old_InterfaceDeclaration_IRootDataObject(
+    use: TypeOnly<old.IRootDataObject>);
+use_old_InterfaceDeclaration_IRootDataObject(
+    get_current_InterfaceDeclaration_IRootDataObject());
+
```

Comment:
These look fine. @ChumpChief [added `IRootDataObject` recently](https://github.com/microsoft/FluidFramework/pull/12351/). Not sure about the other change in this file.